### DISCUSSION TITLE
How to always set session cookie's SameSite attribute to None?

### DISCUSSION THREAD

**Discussion by Cvmcosta** [OpenIDC/mod_auth_openidc#754] [Q&A]
Hello! I have been working on a project that requires me to access an application through an Iframe, for this reason, i need the cookies generated by `mod_auth_openidc` to be **always** created with `SameSite=None`.

Can anyone tell me exactly how to achieve this? I looked around in the documentation and discussions but couldn't find a proper answer.

Thanks in advance!

**Comment by zandbelt** [comment]
recent versions of the module always set `SameSite=None` to its cookies (on https)  by default, no extra config needed for that
